rotkehlchen/tests/utils/dataimport.py

```diff
@@ -74,7 +76,22 @@ def assert_cointracking_import_results(rotki: Rotkehlchen):
     warnings = rotki.msg_aggregator.consume_warnings()
     errors = rotki.msg_aggregator.consume_errors()
     assert len(errors) == 0
-    assert len(warnings) == 4
+    assert len(warnings) == 0
+    mocked_notifier = cast(MockRotkiNotifier, rotki.msg_aggregator.rotki_notifier)
+    assert mocked_notifier.pop_message() == MockedWsMessage(
+        message_type=WSMessageType.CSV_IMPORT_RESULT,
+        data={
+            'source_name': 'Cointracking',
+            'total_entries': 12,
+            'imported_entries': 7,
+            'messages': [
+                {'msg': 'Not importing ETH Transactions from Cointracking. Cointracking does not export enough data for them. Simply enter your ethereum accounts and all your transactions will be auto imported directly from the chain', 'rows': [1, 2], 'warning': True},  # noqa: E501
+                {'msg': 'Not importing BTC Transactions from Cointracking. Cointracking does not export enough data for them. Simply enter your BTC accounts and all your transactions will be auto imported directly from the chain', 'rows': [5], 'warning': True},  # noqa: E501
+                {'msg': 'Unknown asset ADS.', 'rows': [9], 'warning': True},
+                {'msg': 'Staking event for eip155:1/erc20:0xBB0E17EF65F82Ab018d8EDd776e8DD940327B28b(Axie Infinity Shard) at 1641386280 already exists in the DB', 'rows': [12], 'warning': False},  # noqa: E501
```

Comment:
so total is 12, we imported 7, and we see 4 errors. This creates a confusion that what happened with 1. Was it skipped? If yes, then maybe include a message for that as well?

Reply:
Notice that the first entry in `messages` has `'rows': [1, 2]`. So there are in fact 5 errors reported.